src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  background-color: #fff;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
+
+  @extend %text-style;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
i think you don't need two times the same :)

Reply:
yeah )
missed that i've committed an older version